### PR TITLE
[DOCS-6728] - Add validation message

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

File: process-automation/latest/model/authentication.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ The basic properties of creating authentication are:
 | Authentication name | *Required.* The name used to identify the authentication. The name must be in lowercase and be between one and 26 characters in length, for example `token1auth`. |
 | Authentication description | *Optional.* A description of the authentication. This can be your identity provider, for example `Keycloak authentication used`. |
 | Authentication type | *Required.* Select the type of authentication, for example `Basic`. There are three types of authentication you can use, `Basic`, `Client credentials`, and `Bearer token`. |
-| Secured | *Optional.* When selected, credentials are defined during deployment time, or de-select the checkbox to provide them during modeling time. These credentials are used when contacting the external API. Note: All the configuration parameters can be overridden at deployment time. |
```
Comment (Artur-Piechuta) [inline process-automation/latest/model/authentication.md:16]: Are you sure that this sentence is not valid anymore?
_Note: All the configuration parameters can be overridden at deployment time._

image

Comment (oliveralfresco2018) [reply process-automation/latest/model/authentication.md:16]: Morning @Artur-Piechuta 

Yep there is a Note at the bottom of those fields.
image

**Review by gravitonian — APPROVED** [review]
Built and viewed. LGTM.

**Review by Artur-Piechuta — APPROVED** [review]
Looks good to me

**Pull request merged**